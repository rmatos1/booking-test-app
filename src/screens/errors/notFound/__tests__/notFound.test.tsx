import { describe, test } from '@jest/globals';
import { render } from "@testing-library/react";

import { NotFound } from "..";

import { MemoryRouter } from "react-router-dom";


const setup = (): JSX.Element => {
  return(
    <MemoryRouter>
      <NotFound />
    </MemoryRouter>
  )
};

describe('<NotFound />', () => {
  test('should render correctly', () => {
    const wrapper = render(setup());

    expect(wrapper).toMatchSnapshot();
  });
});
