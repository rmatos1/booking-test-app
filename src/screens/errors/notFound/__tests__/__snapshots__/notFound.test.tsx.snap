// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body
    style="overflow: unset;"
  >
    <div>
      <nav
        class="sc-fubDmA jSFNkN"
      >
        <a
          aria-current="page"
          class="sc-pGaPU kpubXZ active"
          href="/"
        >
          New Booking
        </a>
        <a
          class="sc-pGaPU zBnL"
          href="/my-bookings"
        >
          My Bookings
        </a>
      </nav>
      <div
        class="sc-bqyKbq gCCXA-D"
      >
        <img
          alt="not found"
          class="sc-eCstZk cgIxof"
          data-testid="image"
        />
        <div
          class="sc-kfzCjt KTkhM"
        >
          <h1
            class="sc-fKFygU dbnRdB"
          >
            Page not found!
          </h1>
          <p>
            Sorry, it looks like you've lost the path. But we can offer you a little help with some links!
          </p>
          <div
            class="sc-kEjbdu kFWIOM"
          >
            <a
              class="sc-iqHXzD bLPIku"
              href="/"
            >
              New Booking
            </a>
            <a
              class="sc-iqHXzD bLPIku"
              href="/my-bookings"
            >
              My Bookings
            </a>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <nav
      class="sc-fubDmA jSFNkN"
    >
      <a
        aria-current="page"
        class="sc-pGaPU kpubXZ active"
        href="/"
      >
        New Booking
      </a>
      <a
        class="sc-pGaPU zBnL"
        href="/my-bookings"
      >
        My Bookings
      </a>
    </nav>
    <div
      class="sc-bqyKbq gCCXA-D"
    >
      <img
        alt="not found"
        class="sc-eCstZk cgIxof"
        data-testid="image"
      />
      <div
        class="sc-kfzCjt KTkhM"
      >
        <h1
          class="sc-fKFygU dbnRdB"
        >
          Page not found!
        </h1>
        <p>
          Sorry, it looks like you've lost the path. But we can offer you a little help with some links!
        </p>
        <div
          class="sc-kEjbdu kFWIOM"
        >
          <a
            class="sc-iqHXzD bLPIku"
            href="/"
          >
            New Booking
          </a>
          <a
            class="sc-iqHXzD bLPIku"
            href="/my-bookings"
          >
            My Bookings
          </a>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
